.rubocop_todo.yml

```diff
@@ -0,0 +1,187 @@
+# This configuration was generated by
```

Comment:
I don't like optional files in public repos, especially generated files. Please remove.